### ISSUE TITLE
Replication of results from paper

### ISSUE THREAD

**Issue by Naykira** [chamorajg/pytorch_depth_and_motion_planning#1]
Hi @chamorajg, thanks for the uploading pytorch implementation of the 'Unsupervised Monocular Depth Learning in Dynamic Scenes' paper. Were you able to reach the numbers mentioned in the paper for Kitti/Ciyscapes?

**Comment by chamorajg** [comment]
I am currently working on a few videos from KITTI dataset and making some minor fixes to the loss functions. Once completed, I shall also upload the pretrained models on KITTI dataset.

**Comment by Naykira** [comment]
Thanks for the quick reply. Looking forward to the result.
